src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -17,6 +18,8 @@ public static class SarifTransformerUtilities
             { SarifVersion.TwoZeroZero, "sarifv2" }
         };
 
+        public static readonly string[] FullyQualifiedNameDelimiters = { ".", "/", "\\", "::" };
```

Comment:
I'm ok with this for now. Ultimately we might want a command line argument to override this (in which case I'd rename this to `DefaultFullyQualifiedNameDelimiters`. You might do that rename anyway, and add a comment suggesting the possibility of overriding it.

Reply:
Good idea.

Reply:
Renamed